### ISSUE TITLE
Having trouble completing the initial authentication process. 

### ISSUE THREAD

**Issue by WillyKelleher** [NVentimiglia/TDAmeritrade.DotNetCore#1]
Followed instructions, tried every way possible and get Error: Bad Request.

The python tda_api works perfectly, I saw your post in that discord.

**Comment by NVentimiglia** [comment]
That's not good!

I did fight that issue in the past, it was related to malformed json, 

but, the current version is working fine for me. I am even recorded last friday's QQQ activity.

Are you using a fresh pull, I did push some updates friday/

Are you trying the ASP authentication process or the console one?

When pasting the code, I do not filter for the url encoding (the 'code=' part).

**Comment by WillyKelleher** [comment]
I was using the console app to try and get through the initial auth process. I did change the reply url to match my app https://127.0.0.1. Also changed each project to dotnet 4.8 to get it to compile. Had to add a bunch of nuget libraries back as well.

Just tried to upgrade VS to the latest so I could see if dotnet 5 would work and broke VS, nothing will clean or compile and no real messages to go on. This happened last time I upgraded VS so rolling back now. I did have to delete my tda app in the past and recreate so maybe that's it but for now I need to get VS working again before Monday so this is on the back burner.

Thanks but not sure this is worth the time now.

**Comment by WillyKelleher** [comment]
OK so I guess this can be closed. I was able to get the latest version of VS2019 working, not related to this project.
Had to fix the latest VS2019 install by updating config for the following as well as msbuild.
https://docs.microsoft.com/en-us/answers/questions/313193/could-not-start-visual-studio-due-to-an-error-stre.html
https://stackoverflow.com/questions/61323996/could-not-load-file-or-assembly-system-numeric-vectors-in-ml-net

Then I Added nuget package to a fresh clone of this project.
Microsoft.AspNetCore.Mvc.NewtonsoftJson

I updated my tda app to use http://localhost instead of http://127.0.0.1
and it compiles and works as expected.

Thanks 
-Willy
